overlord/devicestate/handlers_install.go

```diff
@@ -111,6 +112,24 @@ func (m *DeviceManager) doSetupRunSystem(t *state.Task, _ *tomb.Tomb) error {
 		return fmt.Errorf("cannot make run system bootable: %v", err)
 	}
 
+	// support dropping cloud-init for grade: dangerous
+	cloudCfg := filepath.Join(dirs.RunMnt, "ubuntu-seed/cloud.cfg.d")
+	if osutil.IsDirectory(cloudCfg) && deviceCtx.Model().Grade() == asserts.ModelDangerous {
+		ubuntuDataCloudCfgDir := filepath.Join(dirs.RunMnt, "ubuntu-data/system-data/etc/cloud/cloud.cfg.d/")
+		if err := os.MkdirAll(ubuntuDataCloudCfgDir, 0755); err != nil {
+			return fmt.Errorf("cannot make cloud config dir: %v", err)
+		}
+		ccl, err := filepath.Glob(filepath.Join(cloudCfg, "*.cfg"))
+		if err != nil {
+			return err
+		}
+		for _, cc := range ccl {
+			if err := osutil.CopyFile(cc, filepath.Join(ubuntuDataCloudCfgDir, filepath.Base(cc)), 0); err != nil {
```

Comment:
What's the behavior of CopyFile if the destination file is already present?

Reply:
It should overwrite the file IIRC